src/pytesmo/time_series/anomaly.py

```diff
@@ -205,8 +229,9 @@ def calc_climatology(Ser,
     else:
         clim_ser = moving_average(clim_ser, window_size=moving_avg_clim, fillna=fillna, min_obs=min_obs_clim)
 
-    clim_ser = clim_ser.reindex(np.arange(366) + 1)
+    clim_ser = clim_ser.reindex(np.arange(n_idx) + 1)
```

Comment:
Can you move this line before the `if wraparound` block? This would solve one of the issues in #250